Packs/Jira/Integrations/JiraEventCollector/JiraEventCollector.py

```diff
@@ -152,8 +152,9 @@ def set_next_run(log: dict) -> dict:
         last_run = demisto.getLastRun()
 
         if not last_run.get('next_time'):
-            last_time = log.get('created', '').removesuffix('+0000')
-            next_time = last_time[:-1] + str(int(last_time[-1]) + 1)
+            last_datetime = log.get('created', '').removesuffix('+0000')
```

Comment:
Will the timezone always be `+0000`?

Reply:
@yaakovi It looks like https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-audit-records/#api-rest-api-3-auditing-record-get-responses